bootstrap.sh

```diff
@@ -298,12 +299,13 @@ install_homebrew() {
   # Update Homebrew.
   export PATH="$HOMEBREW_PREFIX/bin:$PATH"
   logn "Updating Homebrew:"
-  [ -z "$QUICK" ] && brew update --quiet
+  [ -z "$QUICK" ] && brew update
   logk
 
   # On Apple M1 machines we need to add this to the profile
   if [[ "$(uname -m)" == "arm64" ]]; then
     shell_profile=$(get_brew_profile)
+    [ ! -f "${shell_profile}" ] && touch "${shell_profile}"
```

Comment:
This silences the warning grep throws (2 lines below) because the file is missing.